src/control/common/storage/commands.go

```diff
@@ -23,6 +23,7 @@ type StoragePrepareNvmeCmd struct {
 	PCIBlockList string `long:"pci-block-list" description:"Whitespace separated list of PCI devices (by address) to be ignored when unbinding devices from Kernel driver to be used with SPDK (default is no PCI devices)."`
 	NrHugepages  int    `short:"p" long:"hugepages" description:"Number of hugepages to allocate for use by SPDK (default 1024)"`
 	TargetUser   string `short:"u" long:"target-user" description:"User that will own hugepage mountpoint directory and vfio groups."`
+	DisableVFIO  bool   `long:"disable-vfio" description:"Force SPDK to use the UIO driver for NVMe device access"`
```

Comment:
Does there need to be a documentation update for this? When should the admin use it?

Looking at it a different way: Is there a way that we can reasonably reliably detect when the admin would want/need to disable VFIO and just do it for them?

Reply:
doc change PR to follow